Add perl philosophy to ours (#537)
returntocorp/semgrep-docs: docs/contributing/philosophy.md

1. **Run anywhere**<br/>
Semgrep is small (<100MB), has minimal runtime dependencies, and should be easily installable via your programming language or operating system package manager. 

1. **Keep easy things easy, and hard things possible.**<br/>

> Using Semgrep to scan your code, and writing rules with which to scan should be as easy as it can be. We'll also smooth the process with delightful defaults and support every step of the way. But Semgrep's there for you to adapt, and we welcome you using Semgrep in your own custom way. [Hey, there are even examples of scanning cat pictures out there.](https://youtu.be/ybWB2Vf2V50?t=1182)

Reviewer: ```suggestion
Using Semgrep to scan your code, and writing rules with which to scan, should be easy. Semgrep also smooths the process with delightful defaults and support every step of the way. But it’s also adaptable, and we welcome you using Semgrep in your own custom way. Hey, there are even [examples of scanning cat pictures out there](https://youtu.be/ybWB2Vf2V50?t=1182).
```
